src/helpers/create-pr.ts

```diff
@@ -18,20 +18,26 @@ export class CreatePR {
   title = '';
   body = '';
   head?: string;
+  base?: string;
 }
 
-export const createPr = async ({ title, body, head = context.ref.replace('refs/heads/', '') }: CreatePR) => {
-  const {
-    data: { default_branch }
-  } = await octokit.repos.get({ ...context.repo });
+export const createPr = async ({ title, body, head = context.ref.replace('refs/heads/', ''), base }: CreatePR) => {
+  const pr_base = base != null ? base : await getDefaultBranch();
```

Comment:
`base` will be `''` if omitted

```suggestion
  const pr_base = base || await getDefaultBranch();
```